data/ecosystems/m/moonfit.toml

```diff
@@ -0,0 +1,44 @@
+# Ecosystem Level Information
+title = "Moonfit"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/mooni"]
```

Comment:
can you point out how's "Mooni" github org related directly to Moonbeam?

Reply:
Sorry about that, typo, I fixed it to Moonfit, which is a Moonbeam ecosystem fitness tracking dApp: https://moonfit.xyz/